### PR TITLE
MetaT Update

### PR DESCRIPTION
**Pull request by kaijli**
Related to this metaT PR https://github.com/microbiomedata/metaT/pull/36
A bit of a placeholder, but the same successful expanse run `ed1aa6ed-b18c-4e12-85e2-47d1ce569531`. 
@yxu-lanl let me know how the outputs look / what you need / if I should have a finish-metaT task to rename and group all the files.

### CODE REVIEW

**Comment by yxu-lanl** [comment]
1. Rename output folder 'qa' to 'readsQC'.
2. Add stats.json to assembly output folder.
3. Move files in 'mapbak' and 'metat_output' to new folder 'readMapping' and delete the 'mapbak' and 'metat_output' folders.
Screenshot 2024-07-09 at 10 07 31 AM

**Comment by kaijli** [comment]
successful expanse run at `6124aa02-1ddc-4879-a078-fe2336139e2e` with requested edits. @yxu-lanl , please let me know if the output folders look good at `/expanse/projects/nmdc/cromwell/cromwell-executions/metaT/6124aa02-1ddc-4879-a078-fe2336139e2e/call-fi/execution/Metatranscriptomics`

**Comment by yxu-lanl** [comment]
The output folders look good. But <prefix>_stats.json is missing in the annotation folder.
e.g. <prefix>_stats.json:
/expanse/projects/nmdc/edge_app/nmdc-edge/io/projects/RzZAaxkfd84wGmA1/output/Metatranscriptomics/annotation/test_stats.json

File: data/workflow/templates/metaT_wdl.tmpl [hunk 1]
```diff
@@ -1,32 +1,17 @@
...
-    String <WORKFLOW>_informed_by
-    Int <WORKFLOW>_threads
-    File <WORKFLOW>_metat_folder
+    String <WORKFLOW>_strand_type
+   
 
     call <WORKFLOW>.nmdc_metat as <ALIAS> {
         input: 
-        proj=<WORKFLOW>_proj,
+        project_id=<WORKFLOW>_proj,
```
Comment (yxu-lanl) [inline data/workflow/templates/metaT_wdl.tmpl:11]: Should be 'project_id=<WORKFLOW>_project_id'?

**Review by yxu-lanl — APPROVED** [review]

File: data/workflow/templates/metaT_inputs.tmpl [hunk 2]
```diff
@@ -1,14 +1,7 @@
...
-    "main_workflow.<WORKFLOW>_git_url": "https://github.com/microbiomedata/mg_annotation/releases/tag/0.1",
-    "main_workflow.<WORKFLOW>_url_root": "https://data.microbiomedata.org/data/",
-    "main_workflow.<WORKFLOW>_outdir": <OUTDIR>,
-    "main_workflow.<WORKFLOW>_resource": "NMDC-EDGE",
-    "main_workflow.<WORKFLOW>_database": "/expanse/projects/nmdc/refdata/",
-    "main_workflow.<WORKFLOW>_activity_id": <PROJNAME>,
-    "main_workflow.<WORKFLOW>_informed_by": <PROJNAME>,
-    "main_workflow.<WORKFLOW>_threads": 64,
-    "main_workflow.<WORKFLOW>_metat_folder": "/expanse/projects/nmdc/edge_app/test/metaT/"
+    "main_workflow.<WORKFLOW>_strand_type":<STRAND_TYPE>
```
Comment (yxu-lanl) [inline data/workflow/templates/metaT_inputs.tmpl:6]: Need UI for the strand_type? If not, use a default value or delete this line.

**Review by yxu-lanl — CHANGES_REQUESTED** [review]
data/workflow/templates/metaT_inputs.tmpl

File: data/workflow/templates/metaT_inputs.tmpl (hunk 2 above)
Comment (kaijli) [reply data/workflow/templates/metaT_inputs.tmpl:6]: Not sure, i can definitely put a default, but UI would be nice? For NMDC automation it'll be read from metadata

Comment (yxu-lanl) [reply data/workflow/templates/metaT_inputs.tmpl:6]: Use a default for now. We can add UI when needed.

**Review by yxu-lanl — APPROVED** [review]

**Pull request closed without merging**